Example/Podfile.lock

```diff
@@ -120,4 +120,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: ab2b14498bb09c9ba993d5484696be2421070207
 
-COCOAPODS: 1.0.1
+COCOAPODS: 1.1.0.rc.2
```

Comment:
Ahhh. Does this explain the emoji pods? Does that mean they're making a comeback? 

We probably want to stick to `1.0.1` though for now, rather than move over to the RC?